src/main/java/uk/ac/ebi/biostudies/config/IndexConfig.java

```diff
@@ -70,6 +70,9 @@ public class IndexConfig implements InitializingBean, DisposableBean {
     @Value("${index.backup.directory}")
     private String indexBackupDirectory;
 
+    @Value("${index.backup.sync.filename}")
+    private String indexSyncBackupFileName;
+
```

Comment:
Instead of just filename, let's store the absolute file path so that we can update it as needed